looprpc/client.proto

```diff
@@ -213,12 +213,10 @@ message LoopInRequest {
     int64 max_miner_fee = 3;
 
     /**
-    The channel to loop in. If zero, the channel to loop in is selected based
-    on the lowest routing fee for the swap payment from the server.
-
-    Note: NOT YET IMPLEMENTED
+    The last hop to use for the loop in swap. If empty, the last hop is selected
+    based on the lowest routing fee for the swap payment from the server.
     */
-    uint64 loop_in_channel = 4;
+    bytes last_hop = 4;
```

Comment:
While we're at it, I think we should add the `last_hop` (and maybe the `external`) fields to the `SwapStatus` message. Otherwise a user will never be able to see this again through the command line or REST.

Reply:
I looked into what this takes, but it isn't as easy as adding the fields in the proto and populating it. The quick fix would be to add loop-type-specific fields like these to `SwapInfo`. This struct however is only supposed to contain the common fields. Having a growing number of unused fields (unused in the other swap type) isn't good for maintainability long term. 

I think it is better to more thoroughly think through what direction we want to go with this and consider exposing the original swap parameters as a separate feature and pr.

Reply:
Okay, that sounds good to me. There are other fields that are currently not exposed through the status calls so it makes sense to add them all at once in a follow-up PR.